corehq/apps/userreports/app_manager.py

```diff
@@ -32,7 +32,7 @@ def get_case_data_source(app, case_type):
         display_name=case_type,
         configured_filter=make_case_data_source_filter(case_type),
         configured_indicators=[
-            make_case_property_indicator(property) for property in property_builder.get_properties(case_type)
+            make_case_property_indicator(property) for property in prop_map[case_type]
```

Comment:
Could be 
    
     configured_indicators=map(make_case_property_indicator, prop_map[case_type])